transaction/interfaces.py

```diff
@@ -22,12 +22,19 @@ class ITransactionManager(Interface):
     """
 
     def begin():
-        """Begin a new transaction.
+        """Explicitly begin a new transaction.
 
         If an existing transaction is in progress, it will be aborted.
 
-        The newTransaction() method of registered synchronizers is called,
+        The ``newTransaction`` method of registered synchronizers is called,
         passing the new transaction object.
+
+        Note that transactions may be started implicitly without
+        calling ``begin``. In that case, ``newTransaction`` isn't
```

Comment:
"without calling ``begin``, for example, by calling ``get`` when there is no transaction in progress."

That's not really right, either, though, because what I think you're referring to is different than a Transaction as managed by a manager. See my comment below.